### ISSUE TITLE
Exception nanopass incompatible fasl-object version 8.4

### ISSUE THREAD

**Issue by ChlorophyII** [cisco/ChezScheme#145]
I tried to follow  README.md under ~/nanopass/lib to use (compile-file). But when I (import (nano pass)), it complained the following:

```
@XX-MBP:ChezScheme-master $ petite --libdirs .:/XXX/ChezScheme-master/nanopass/lib/csv8.4/a6osx 
Petite Chez Scheme Version 9.4.1
Copyright 1984-2016 Cisco Systems, Inc.

> (import (nanopass))
Exception: incompatible fasl-object version 8.4 found in /XXX/ChezScheme-master/nanopass/lib/csv8.4/a6osx/nanopass.so
Type (debug) to enter the debugger.
```
MBP 2014 mid
macOS Sierra version 10.12.3
Chez Scheme version 9.4.1

I searched on google but found no valuable info. Any help?

**Comment by jltaylor-us** [comment]
The exception means that the `(nanopass)` library was compiled with a different version of Chez Scheme (or Petite Chez Scheme in your example) than the one you are currently running (and the file formats used by the two different versions are not binary-compatible).
If there's something confusing in the nanopass docs then @akeep should be able to clear it up, but in the mean time I can only suggest that you rebuild your `(nanopass)` library from source, making sure that your `PATH` is set correctly to pick up the current version of `scheme`.

**Comment by ChlorophyII** [comment]
Thank you for your info @jltaylor-us.  I have solved this problem by manually moving 
XXX/ChezScheme-master/nanopass/lib/csv8.4/a6osx/nanopass.so
to
/usr/local/lib/csv9.4.1/a6osx/nanopass.so

This is quite confusing. I thought after I compiled Chez Scheme, everything would be well prepared. I could only find nanopass.so file under the directory /csv8.4. (There is no csv9.4 in the folder of source files of Chez Scheme.) So I thought following the README file would solve this problem, and 8.4 and 9.4 nanopass are compatible. Actually, they are, the problem is the directory. Maybe something need to be updated?@akeep
